tests/cupyx_tests/scipy_tests/signal_tests/test_peak_finding.py

```diff
@@ -0,0 +1,394 @@
+import pytest
+
+import cupy
+from cupy.cuda import driver
+from cupy.cuda import runtime
+from cupy import testing
+import cupyx.scipy.signal  # NOQA
+
+import numpy as np
+
+try:
+    import scipy
+    import scipy.signal  # NOQA
+except ImportError:
+    scipy = None
```

Comment:
We'd like to avoid introducing the name `scipy` here and use `for`-loop instead where `scipy` is used in `pytest.mark.parametrize`.